pkgs/applications/misc/gsctl/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "gsctl";
+  name = "${pname}-${version}";
+  version = "0.15.1";
+
+  goPackagePath = "github.com/giantswarm/gsctl";
+
+  src = fetchFromGitHub {
+    owner = "giantswarm";
+    repo = "${pname}";
+    rev  = "${version}";
+    sha256 = "151w8myqxw86vprpzi4hg9dd3s40lr2y1pl4g8xqi21dxlmfz8mx";
+  };
+
+  buildFlagsArray = ''
+    -ldflags=-X github.com/giantswarm/gsctl/config.Version=${version}
+    -X github.com/giantswarm/gsctl/config.BuildDate=2019-05-09T09:00:00Z
```

Comment:
Setting a build date like this is impure. can we set it to unix epoch?

Reply:
This is actually broken in latest version, so removed for now. Will re-add (with epoch) in next version.